isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/referenzarchitektur/inhalt.adoc

```diff
@@ -468,6 +468,45 @@ Für derartige Aufrufe gibt es keine Vorgaben: Sie können aus beliebigen Schich
 [id="image-CallFAmGuX",reftext="{figure-caption} {counter:figures}"]
 image::blaupausen:referenzarchitektur/CallFAmGuX.png[align="center"]
 
+[[zentrale-sicherheitsaspekte]]
+=== Zentrale Sicherheitsaspekte in der Referenzarchitektur
+
+Die IT-Sicherheit ist insbesondere dann für Softwaresysteme eine wichtige Anforderung, wenn diese sensible oder gar personenbezogene Daten verarbeiten.
+Solche Systeme stellen hohe Anforderungen an Vertraulichkeit, Integrität und Verfügbarkeit.
+Eine Referenzarchitektur muss als grundlegendes Element der IT-Sicherheitsstrategie die Einhaltung solcher Schutzziele unterstützen.
+
+Ein Pfeiler der IT-Sicherheitsstrategie ist der Schutz der Daten und IT-Systeme vor unberechtigter Nutzung.
+Die IsyFact Referenzarchitektur legt fest, dass jegliche Nutzung von IsyFact-Anwendungen nur authentifizierten und dafür autorisierten Benutzern möglich ist.
+Diese Forderung wird im Querschnitt-Baustein Security umgesetzt, den alle IsyFact-Anwendungen zwingend bei der Kommunikation, sowohl mit anderen Anwendungen innerhalb der Anwendungslandschaft, als auch mit Systemen außerhalb und auch bei der Kommunikation mit Nutzern, einsetzen müssen.
```

Comment:
Die Kommunikation zwischen Systemen ist an dieser Stelle erst einmal nicht relevant und würde ich entfernen. 
Die Bausteinbeschreibung würde ich auch weiter nach hinten stellen. Wichtig ist hier zunächst zu erläutern:
1.) Für die Authentifizierung wird ein zentraler Service (IAM-Service) in der Anwendungslandschaft genutzt. 
2.) Die Zugriffswegen sind entweder 1.) über das Portal für Anwender, 2a) über die Service-Gateways für externe Systeme, 2b) über das Authentication-Gateway für externe Systeme und 3) direkt am IAM-Service für interne Systeme einer Anwendungslandschaft. Diese Zugriffswegen müssen grob beschrieben werden (kein OAuth2 Spezifika) und dann zu den relevanten Dokumenten verlinkt werden.
3.) Die Autorisierung findet an jeder Systemgrenze statt. Je nach Schutzbedarf und Anforderung kann auch innerhalb einer IT-Systems eine zusätzliche Autorisierung stattfinden. 
4.) Für die Bausteinbeschreibung würde ich einfach nur in wenigen Sätzen erwähnen, dass die Funktionen zur Authentifizierung und Autorisierung durch den Baustein "Security" umgesetzt werden. Dies gilt nicht für die Authentifizierung am Portal, die durch das Portal umgesetzt wird (hier wird der Baustein isy-security nicht verwendet), sondern der Portal Apache greift direkt auf den IAM-Service zu.